scripts/audit_ru_translations.py

```
@@ -0,0 +1,370 @@
+    errors = [
+        label
+        for field, label in labels.items()
+        if getattr(expected, field) != getattr(actual, field)
+    ]
+    metadata_surface = "\n".join(target_prose.splitlines()[:15])
+    if re.search(r"\*\*[A-Z][A-Za-z ]+:\*\*", metadata_surface):
+        errors.append("visible English metadata labels")
+    if re.search(r"\b(?:Phase|Lessons?|lesson|lessons|minutes?|hours?)\b", metadata_surface):
+        errors.append("visible English metadata values")
+    for alt_text in re.findall(r"!\[([^\]\n]+)\]\(", target_prose):
+        if not re.search(r"[А-Яа-яЁё]", alt_text) and len(re.findall(r"[A-Za-z]{2,}", alt_text)) >= 3:
```

Ruff (0.16.1)

[warning] String contains ambiguous `А` (CYRILLIC CAPITAL LETTER A). Did you mean `A` (LATIN CAPITAL LETTER A)?

(RUF001)